### PR TITLE
Add Codespaces badge

### PR DESCRIPTION
**Pull request by Alkarex**
Follow-up of https://github.com/FreshRSS/FreshRSS/pull/4860 , https://github.com/FreshRSS/FreshRSS/pull/4859
Discussion https://github.com/FreshRSS/FreshRSS/discussions/4861

With slight rework of some elements of the readme around

### CODE REVIEW

File: README.fr.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 # FreshRSS
 
-FreshRSS est un agrégateur de flux RSS à auto-héberger à l’image de [Leed](https://github.com/LeedRSS/Leed) ou de [Kriss Feed](https://tontof.net/kriss/feed/).
```
Comment (Alkarex) [inline README.fr.md:8]: The link https://tontof.net/kriss/feed/ was dead, and not super relevant anymore. To be meaningful for newcomers, we should either give a couple of examples of more well-known systems such as Feedly, or have a list somewhere in the documentation, or link to e.g. https://alternativeto.net/software/freshrss/

Comment (aledeg) [reply README.fr.md:8]: The link is dead for now. See https://web.archive.org/web/20221104140019/https://tontof.net/index.php?2022/11/04/11/04/00-sobriete-choisie.
We should keep it for now and reevaluate later.

Comment (aledeg) [reply README.fr.md:8]: The project is still available https://github.com/tontof/kriss_feed.
We might just change the link target.

Comment (Alkarex) [reply README.fr.md:8]: It is mostly that I do not find it relevant (anymore), at least not in an intro. My reasoning is that this paragraph is to explain what FreshRSS is, and the likelihood of newcomers to be enlightened by naming Leed or Kriss is probably near zero.
If we are to give an example, naming Feedly (or even Google Reader) might be more helpful.
If relevant, we could have a new section further down or in the documentation with some selected alternatives.

**Pull request merged**